ipaplatform/base/paths.py

```diff
@@ -49,7 +49,7 @@ class BasePathNamespace(object):
     HTTPD_IPA_CONF = "/etc/httpd/conf.d/ipa.conf"
     HTTPD_NSS_CONF = "/etc/httpd/conf.d/nss.conf"
     HTTPD_SSL_CONF = "/etc/httpd/conf.d/ssl.conf"
-    IPA_KEYTAB = "/etc/httpd/conf/ipa.keytab"
+    HTTP_KEYTAB = "/etc/gssproxy/http.keytab"
```

Comment:
I don't see code to remove and invalidate the old keytab ```/etc/httpd/conf/ipa.keytab```. In case the principal should be disabled or invalidated, you can't use```ipa-rmkeytab```. It does not invalidate the principal on the server side.